keanu-python/docs/keanu.vertex.rst

```diff
@@ -1,6 +1,36 @@
 keanu.vertex
 ------------
-This package exposes all of Keanu's vertices which can be used to construct Bayesian Networks.
+
+Distributions
+~~~~~~~~~~~~~
+
+* :meth:`keanu.vertex.Bernoulli`
+* :meth:`keanu.vertex.Beta`
+* :meth:`keanu.vertex.Cauchy`
+* :meth:`keanu.vertex.ChiSquared`
+* :meth:`keanu.vertex.Dirichlet`
+* :meth:`keanu.vertex.Exponential`
+* :meth:`keanu.vertex.Gamma`
+* :meth:`keanu.vertex.Gaussian`
```

Comment:
Would you be able to add in `Geometric` distribution to this list?

Reply:
Yup. This really should be automated somehow

Reply:
I agree, there is a task on the backlog for this.